pkg/inject/sidecar_test.go

```diff
@@ -187,7 +187,7 @@ func TestSidecarDefaultPorts(t *testing.T) {
 func TestSkipInjectSidecar(t *testing.T) {
 	// prepare
 	jaeger := v1.NewJaeger(types.NamespacedName{Name: "TestSkipInjectSidecar"})
-	dep := dep(map[string]string{Annotation: "non-existing-operator"}, map[string]string{})
+	dep := dep(map[string]string{}, map[string]string{Label: "non-existing-operator"})
```

Comment:
Depending on the outcome of the previous comment around the test `TestInjectSidecar`, this test might now be ambiguous: is this testing that the sidecar is being skipped because there's no `Annotation` set, or is it because the instance isn't existing?